src/components/Details/ReservationModal.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+
+const ReservationModal = ({ visible, onOk, onCancel }) => {
+  const [reservationDate, setReservationDate] = useState(new Date());
+  const handleVisible = () => {
+    if (visible) return 'block ';
+    else return 'hidden ';
+  };
+
+  // todo: Get data stored in redux store
+  const city = 'Los Angeles - Dummy';
+  const hotel = 'Luxury hotel - Dummy';
+  const hotelRoom = 'Honeymoon Special - Dummy';
+
+  return (
+    <div
+      className={
+        handleVisible() + 'bg-black/20 absolute top-0 left-0 w-screen h-screen'
```

Comment:
That's a clever implementation of adding the visible class to your modal window 👍